lua/entities/gmod_wire_egp/lib/objects/text.lua

```diff
@@ -35,12 +33,10 @@ if CLIENT then
 	cam_PushModelMatrix = cam.PushModelMatrix
 	cam_PopModelMatrix = cam.PopModelMatrix
 	mat = Matrix()
-	mat:Scale(Vector(1, 1, 1))
 	matAng = Angle(0, 0, 0)
-	matTrans = Vector(0, 0, 0)
 end
 
-Obj.Draw = function( self )
+Obj.Draw = function( self, ent )
```

Comment:
`function Obj:Draw(ent)`

Reply:
Didn't rewrite it because that's how every method in the egp libs are defined. If I change this one, should I just make a commit to update the styling in this file?